src/librustdoc/html/static/search.js

```diff
@@ -354,10 +357,13 @@ window.initSearch = function(rawSearchIndex) {
                 if (literalSearch) {
                     if (val.generics && val.generics.length !== 0) {
                         if (obj.length > GENERICS_DATA &&
-                              obj[GENERICS_DATA].length >= val.generics.length) {
+                              obj[GENERICS_DATA].length >= 0) {
```

Comment:
Should this be:

```suggestion
                             obj[GENERICS_DATA].length > 0) {
```

?